src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const x = new Big(numberOne);
+  const y = new Big(numberTwo);
+  let result;
+  switch (operation) {
+    case '+':
+      result = x + y;
+      break;
+    case '-':
+      result = x - y;
+      break;
+    case '÷':
+      result = x / y;
+      break;
+    case 'x':
+      result = x * y;
+      break;
+    case '%':
+      result = x / 100;
+      break;
+    default:
+      return;
```

Comment:
Please make sure that you use functions like plus, minus, times, div, and modifier provided by Big.js